awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/AwsApplicationSignalsCustomizerProvider.java

```diff
@@ -80,6 +80,9 @@ public class AwsApplicationSignalsCustomizerProvider
       "otel.aws.app.signals.enabled";
   private static final String APPLICATION_SIGNALS_ENABLED_CONFIG =
       "otel.aws.application.signals.enabled";
+
+  private static final String OTEL_RESOURCE_PROVIDERS_AWS_ENABLED =
+      "otel.resource.providers.aws.enabled";
```

Comment:
is it new for v2 only or for both latest v1 as well?

Reply:
It is new for V2 only. This PR will only be merged after the V1 version is branched out to `releases/1.33.x `branch. So the code change in this PR will only apply to ADOT Java v2.x code base.